src/msc/msc.csproj

```diff
@@ -1,7 +1,11 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <IsTool>true</IsTool>
     <OutputType>Exe</OutputType>
+    <IsPackable>false</IsPackable>
+    <ProjectName>Minsk</ProjectName>
+    <RootNamespace>Minsk</RootNamespace>
```

Comment:
`IsTool` property is only useful when packing but it can also be used to customize our own build.

We are using `ProjectName` that is defined in the common targets, to publish the tools into a common folder under publish directory.

If unset, the `RootNamespace` will be `msc`. So, we set it explicitly.